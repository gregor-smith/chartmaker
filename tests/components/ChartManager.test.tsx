test.todo('renders chart selector and buttons')
