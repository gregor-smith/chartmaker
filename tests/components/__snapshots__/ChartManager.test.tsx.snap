// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders chart selector and buttons 1`] = `
.emotion-1:not(:last-of-type) {
  border-bottom: 1px solid white;
  padding-bottom: 1rem;
  margin-bottom: 1rem;
}

.emotion-0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

<div
  id="test-render-container"
>
  <div
    class="emotion-1"
  >
    <div
      class="mock-chart-selector"
    >
      Charts: [{"name":"test chart 1","rowsX":5,"rowsY":5,"shape":{"tag":"Collage"},"albums":[]},{"name":"test chart 2","rowsX":10,"rowsY":10,"shape":{"tag":"Top","size":100},"albums":[]},{"name":"test chart 3","rowsX":10,"rowsY":10,"shape":{"tag":"Top","size":42},"albums":[]}]
      Active chart index: 1
    </div>
    <div
      class="emotion-0"
    >
      <div
        class="mock-new-chart-button"
      />
      <div
        class="mock-rename-active-chart-button"
      />
      <div
        class="mock-delete-active-chart-button"
      />
    </div>
  </div>
</div>
`;
